packages/lifecycle/test/fixtures/issue-3907/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "issue-3907",
+  "version": "1.0.0",
+  "scripts": {
+    "echo": "node echo.sh"
```

Comment:
We can't use `| json-append output.json` here.

The arguments will append to `json-append` not the real command, so we use one customized shell to do that.

Reply:
it is OK to cover it with a unit test, if it is easier that way.